gui/packages/desktop/src/main/index.js

```diff
@@ -454,10 +512,6 @@ const ApplicationMain = {
       fullscreenable: false,
       show: false,
       frame: false,
-      webPreferences: {
-        // prevents renderer process code from not running when window is hidden
-        backgroundThrottling: false,
```

Comment:
FYI background throttling is on by default and allows to throttle updates to browser window when it's in background.